recipe/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+xorg-x11-server-Xorg
```

Comment:
For 8.0a3 we definitely want to make optional dependencies optional again as a dependency on x11 really isn't good.

Reply:
Yeah, that is not good.

Reply:
I think this is for build only. From the `[`yum_requirements.txt](https://conda-forge.org/docs/maintainer/knowledge_base.html#yum-deps) docs:

>Dependencies can be installed into the build container with yum, by listing package names line by line in a file named yum_requirements.txt in the recipe directory of a feedstock.
>
>There are only very few situations where dependencies installed by yum are acceptable. These cases include
>
>        satisfying the requirements of CDT packages during test phase
>        installing packages that are only required for testing

But once we find a solution for optional dependencies this shouldn't be an issue any more I think.